### PR TITLE
SYN-3238: Update certificates section to be more clear

### PR DESCRIPTION
**Pull request by sangn-splunk**
These instructions really apply for API and Uptime tests, so I'm just modifying these instructions to be more clear around that.

**Requirements**

- [x] Content follows Splunk guidelines for style and formatting.
- [x] You are contributing original content.

**Describe the change**

Enter a description of the changes, why they're good for the Observability Cloud documentation, and so on.

If this contribution is time sensitive, tell us when you'd like this PR to be merged.

### CODE REVIEW

File: synthetics/test-config/private-locations.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ Follow these steps to limit logging:
 
 Adding certificates in Synthetics
```
Comment (gschatz-splunk) [inline synthetics/test-config/private-locations.rst:116]:
```suggestion
Add certificates in Synthetics
```

Comment (sangn-splunk) [reply synthetics/test-config/private-locations.rst:116]: Hi @gschatz-splunk! I can make this change, but it is pre-existing in docs: https://docs.splunk.com/observability/en/synthetics/test-config/private-locations.html#adding-certificates-in-synthetics

Would you still want me to proceed?

File: synthetics/test-config/private-locations.rst [hunk 2]
```diff
@@ -130,6 +130,7 @@ For example, here is what a command might look like after you modify it to fit y
 
     docker run -e "RUNNER_TOKEN=<insert-token>" --volume=`pwd`/certs:/usr/local/share/ca-certificates/my_certs/ quay.io/signalfx/splunk-synthetics-runner:latest bash -c "sudo update-ca-certificates && bundle exec bin/start_runner"
 
+**Note for Browser tests:** When using custom root CA certificates, please be aware that they are not compatible with Browser tests. Browser tests require SSL/TLS validation for accurate testing. You do have the option to disable SSL/TLS validation for browser tests when necessary, but custom root CA certificates are not supported for this test type.
```
Comment (gschatz-splunk) [inline synthetics/test-config/private-locations.rst:133]:
```suggestion

.. Note:: Custom root CA certificates aren't supported for Browser tests. Browser tests require SSL/TLS validation for accurate testing. Optionally, you can deactivate SSL/TLS validation for Browser tests when necessary.



```

**Comment by gschatz-splunk** [comment]
hi! @sangn-splunk thanks so much for your contribution and for being such a great friend of the docs! Please keep the contributions coming :) 

I left two small suggestions for you to peruse at your leisure and if you are happy with them we can go ahead and merge it in :)

File: synthetics/test-config/private-locations.rst (hunk 1 above)
Comment (gschatz-splunk) [reply synthetics/test-config/private-locations.rst:116]: yep! I added it since i was in there and we had the opportunity to change it ;)

**Review by gschatz-splunk — APPROVED** [review]

**Comment by gschatz-splunk** [comment]
merged, thanks! :)

**Pull request merged**